docs/product-assembly_case-management-getting-started/src/main/asciidoc/case-management-designing-a-case-proc.adoc

```diff
@@ -1,2 +1,99 @@
-[id='']
-= TODO
\ No newline at end of file
+[id='case-management-designing-a-case-proc']
+= Designing a case
+
+Cases are designed using the *Case definition* design editor in {CENTRAL}. The case design is the basis of case management and sets out the specific goals and tasks for each case. The design can be added to and modified dynamically during runtime.
+
+To demonstrate how to design a case, {PRODUCT} includes the IT Orders sample project in {CENTRAL}, which includes the following case definition:
+
+image::itorders-orderhardware-process.png[IT Orders case design]
+
+You can refer to and interact with this case definition by importing the sample project in {CENTRAL}. 
+
+.Prerequisite 
+* You have followed the procedure in <<case-management-creating-a-case-proc>> to create a new case in {CENTRAL}.
+
+.Procedure
+. *Create a new case definition.*
++
+In the *Assets* window, click *Add Asset* in the upper-right corner of the project perspective and select *Case definition*. 
+
+. Complete the required information in the *Create new Case definition* window:
++
+* `name` - the name of the case. This is usually the subject of the case or project that is being case managed.
+* `package` - the location in which case file is to be created.
+* `Case ID prefix (optional)` - a configurable prefix that allows you to easily distinguish different types of cases. The prefix is follows with a generated ID in the format `ID-XXXXXXXXXX`, where `X` is a generated number that provides a unique ID for the case instance. If a prefix is not provided, the default prefix is `CASE` and generates instances with the following identifiers: 
++
+`CASE-0000000001`
++
+`CASE-0000000002`
++
+`CASE-0000000003`
++
+and so on. If a prefix is set to something else, such as `IT` for example, the following identifiers are generated:
++
+`IT-0000000001`
++
+`IT-0000000002`
++
+`IT-0000000003`
+
++
+. Click *Ok* to open the case designer.
++
+The designer creates a `WorkDefinition` file that includes the work items required for case definition. These appear as usable objects in the case designer *Object Library*, accessible by clicking the image:3897.png[left arrows] in the upper-left corner of the designer.
++ 
+** Verify that the case designer includes the `Milestones` and `Cases` objects by either checking the *Object Library* dock on the left side of the case designer, or by checking the `WorkDefinition` file under *Work Item Definitions* in the *Project Explorer* window on the left side of the {CENTRAL} workbench.
```

Comment:
Here and elsewhere (comes up a few times), I'd just make this a step. If not, flush left no bullet.